srcpkgs/xfce4-panel-profiles/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'xfce4-panel-profiles'
+pkgname=xfce4-panel-profiles
+version=1.0.14
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --python=python3"
+hostmakedepends="intltool tar"
```

Comment:
Does it need tar?

Reply:
I tested in a new branch, cleaned the cache via `./xbps-src bootstrap-update`, then building it w/o tar just failed, so yes, it does need tar. It's required to build locales judging from [Makefile](https://gitlab.xfce.org/apps/xfce4-panel-profiles/-/blob/master/Makefile.in.in#L37-56).

Reply:
~~Okay, second look on that it seems to get rid of intltool actually, I'll update that.~~ Never mind, 1.0.14 does not include the change so intltool is still needed.